internal/pipeline/file_receiver.go

```diff
@@ -186,6 +186,7 @@ func (fr *FileReceiver) RegisterAdminRoutes(r *admin.Server) {
 // handleMessage will listen for an incoming.ACHFile to pass off to an aggregator for the shard
 // responsible. It does so with a database lookup and the fixed set of Shards from the file config.
 func (fr *FileReceiver) handleMessage(ctx context.Context, sub stream.Subscription) chan error {
+	// TODO: This locking does not appear to serve any purpose.
```

Comment:
Should it be removed if it doesn't do anything?

Reply:
Yes, I put the comment so that we take care of it as some point, but it is not highly relevant to this PR, and I don't think that it impacts anything.